wheeltec:
    hardware_interface:
        loop_hz: 10 # hz
        joints:
            - left_finger_joint
